docs/source/modules/search/citrination_client.search.core.query.rst

```diff
@@ -0,0 +1,45 @@
+citrination\_client.search.core.query package
```

Comment:
make sure that all the classes in these modules have their properties documented (in addition to the methods)